rollup.config.js

```diff
@@ -4,23 +4,21 @@ import graphql from '@rollup/plugin-graphql';
 import localResolve from 'rollup-plugin-local-resolve';
 import nodeResolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
-import pkg from './package.json';
+import { visualizer } from 'rollup-plugin-visualizer';
 
-let FORMAT = process.env.FORMAT;
+const FORMAT = process.env.FORMAT;
 
-// graphql-tools currently has a rollup build failure, so always call it an external until they fix it
```

Comment:
I have added back `graphql-tools` as default dependancy since we are not using it in zm-x-web. So it will be bundled again with the api-js-client. I did not see any failure with the build. @silentsakky let me know if this change is fine.

Reply:
change looks good but I have a question, how are we notifying the package user that there are certain runtime dependencies which needs to be satisfied to make it work. in normal scenarios we use peerDependencies for that but I am not sure we can do that here.

Reply:
Moving the dependancies form dependancy array to peerDependancies will result in build failure in our case, since it is not supported below npm version 7. So in that case we must add them in the devDependancies or keep then in the dependancies as it is. I though it will be confusing to keep the same packages at two different locations to serve same purpose.